bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -56,6 +56,7 @@ spec:
           trap quit SIGTERM
           /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --system-id=random
           ovs-appctl vlog/set "file:${OVS_LOG_LEVEL}"
+          /usr/share/openvswitch/scripts/ovs-ctl --protocol=udp --dport=6081 enable-protocol
```

Comment:
A bit confused - shouldn't we template in the value from the network operator config?

Reply:
@casey, this PR is not related to the genevePort configuration, PR 341 (which is blocked on PR 333), it is related to PR 329.
I will add the template in PR 341 when the genevePort is configurable. Adding this directly to PR 341 is fixing a problem that is not part of PR 341.

Reply:
Gotcha. For my understanding, what does it fix?

Reply:
@squeed it opens the port in systems that use iptables (such as bare metal RHEL). In this case its the geneve port. Elsewhere in ovnkube.sh it opens the sdb(6642)/ndb(6641) ports.